Makefile

```diff
@@ -58,38 +58,39 @@ oss:
 JQ := $(shell command -v jq 2> /dev/null)
 RNVERSION=$(shell cd node_modules/react-native && pod ipc spec React.podspec | jq '.version' -r)
 YOGAVERSION=$(shell cd node_modules/react-native/ReactCommon/yoga && pod ipc spec yoga.podspec | jq '.version' -r)
-FOLLYVERSION=$(shell cd node_modules/react-native/third-party-podspecs && pod ipc spec Folly.podspec | jq '.version' -r)
-DOUBLECONVERSIONVERSION=$(shell cd node_modules/react-native/third-party-podspecs && pod ipc spec DoubleConversion.podspec | jq '.version' -r)
-GLOGVERSION=$(shell cd node_modules/react-native/third-party-podspecs && pod ipc spec glog.podspec | jq '.version' -r)
 YOGA_SRC_BEFORE=yoga\/\*\*\/\*.{c,h}
 YOGA_SRC_AFTER=ReactCommon\/yoga\/yoga\/\*\*\/\*.\{c,h\}
+YOGA_MASTER_PODSPEC_URL = "https://raw.githubusercontent.com/facebook/react-native/master/ReactCommon/yoga/yoga.podspec"
 
 update_specs_repos:
 ifndef JQ
 		$(error "Please install jq before running `brew install jq`")
 endif
 
+	# Comment out Yoga, as it's dealt with below in ship_yoga
+
 	@echo "Updating Artsy specs repo";
 	pod repo update artsy
 
 	@echo "Creating folder in artsy specs repo";
 	mkdir ~/.cocoapods/repos/artsy/React/$(RNVERSION)
-	mkdir ~/.cocoapods/repos/artsy/yoga/$(YOGAVERSION)
-	mkdir ~/.cocoapods/repos/artsy/Folly/$(FOLLYVERSION)
-	mkdir ~/.cocoapods/repos/artsy/DoubleConversion/$(DOUBLECONVERSIONVERSION)
-	mkdir ~/.cocoapods/repos/artsy/glog/$(GLOGVERSION)
```

Comment:
This code was wrong, we didn't need to ship them to our specs repo because they existed inside the public specs repo